modules/installation-aws-config-yaml.adoc

```diff
@@ -254,12 +269,12 @@ endif::restricted[]
 
 
 ----
-ifndef::gov,china[]
+ifndef::secret,china[]
```

Comment:
These should include `gov` as the installer survey can only be used in the public partition, not the government partition.

Reply:
Done.